modules/apps/29-fee/types/expected_keepers.go

```diff
@@ -14,12 +14,15 @@ type AccountKeeper interface {
 	GetAccount(sdk.Context, sdk.AccAddress) types.AccountI
 }
 
+// ICS4Wrapper defines the expected ICS4Wrapper for middleware
+type ICS4Wrapper interface {
+	SendPacket(ctx sdk.Context, channelCap *capabilitytypes.Capability, packet ibcexported.PacketI) error
```

Comment:
We should also add `WriteAcknowledgement` here.

Reply:
updated :+1: